PathOfBuildingCommunity/PathOfBuilding
# Remove duplicated basename

Fixes #6513 .

### Description of the problem being solved:
Duplication of basename in exports

### Steps taken to verify a working solution:
- Load a broken build, make a change, eg add a node to the tree
- Save the build and look in the .xml.
- Confirm there is no duplication of basename

### Link to a build that showcases this PR:
None required. Happens all the time

### Before screenshot:
![image](https://github.com/PathOfBuildingCommunity/PathOfBuilding/assets/4302241/b0669a4b-688c-460c-9a9e-8f2cd30d0817)

### After screenshot:
![image](https://github.com/PathOfBuildingCommunity/PathOfBuilding/assets/4302241/9c8c98a3-35fb-476e-99af-1230be2fb89b)
